docs/csharp/language-reference/operators/index.md

```diff
@@ -57,7 +57,7 @@ These are the highest precedence operators.
 
 [nameof](nameof.md) - obtains the simple (unqualified) name of a variable, type, or member as a constant string.
 
-[delegate](../../programming-guide/statements-expressions-operators/anonymous-methods.md) – declares and returns a delegate instance.
```

Comment:
@BillWagner though this programming guide article provides more trivia (which can found in the spec) about the anonymous methods, I suggest to remove it and redirect to the newly created article about the `delegate` operator. So, that one to remove:
https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/statements-expressions-operators/anonymous-methods

If that's fine with you, I'll do that in the separate PR.

Reply:
I like that idea. We should minimize coverage of using the `delegate` syntax. It's not preferred anymore.